lib/campaign/CampaignHandler.cpp

```diff
@@ -50,12 +50,14 @@ void CampaignHandler::readCampaign(Campaign * ret, const std::vector<ui8> & inpu
 	{
 		JsonNode jsonCampaign(reinterpret_cast<const std::byte*>(input.data()), input.size(), filename);
 		readHeaderFromJson(*ret, jsonCampaign, filename, modName, encoding);
+		ret->overrideCampaign();
```

Comment:
Shouldn't campaigns in json format already contain complete data? Reason why we have these overrides is due to some of hardcoded logic in h3 campaigns

Reply:
Yes, you're right.